doc/neps/nep-0018-array-function-protocol.rst

```diff
@@ -276,8 +327,8 @@ to be explicitly recognized. Libraries like Dask, CuPy, and Autograd
 already wrap a limited subset of SciPy functionality (e.g.,
 ``scipy.linalg``) similarly to how they wrap NumPy.
 
-If we want to do this, we should consider exposing the helper function
-``do_array_function_dance()`` above as a public API.
+If we want to do this, we should expose the helper function
+``try_array_function_override()`` as a public API.
```

Comment:
More importantly, the wrapper should be exposed, and the annotation API agreed upon.